### ISSUE TITLE
Inconsistent input frame count calculation during interpolation

### ISSUE THREAD

**Issue by anuragl94** [n00mkrad/flowframes#89]
I have a 24fps 60s webm file as input, so that's 1440 frames. However, when I interpolate it, it's miscalculating the input frame count.
![image](https://user-images.githubusercontent.com/9764372/146265375-4863bfd3-0036-4155-97ac-b8fce5539b56.png)
![image](https://user-images.githubusercontent.com/9764372/146265477-2aa7f02a-bf48-459f-b1f3-1f983aaf6f46.png)
![image](https://user-images.githubusercontent.com/9764372/146265600-28cb705d-fbc6-4101-8dda-eac7eb0cb111.png)

Please note that during these runs, I haven't even changed the input. I simply press the interpolate button after the last operation completes.

**Comment by n00mkrad** [comment]
You mean it should be 1797 frames instead of 155/277/135?

**Comment by anuragl94** [comment]
That's right. Also, it's important to mention that the output video is still of the same length as the input (as expected). This indicates that the program is able to accurately calculate input/output fps and render the interpolated frames correctly. It just stops interpolating frames after an arbitrary point.

**Comment by Nuzza** [comment]
I have the exact same issue. I am trying to increase the fps of a video that is quite long, Flowframes detects the total number of frames, however, stops interpolating after the first 1000 frames. Once it's finished, I can open the video fine and it starts playing normally, however, after a certain time, the video just stops, but the audio continues playing like normal for the remaining length of the video.
